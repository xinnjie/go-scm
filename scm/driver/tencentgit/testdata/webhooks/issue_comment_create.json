{
  "object_kind": "note",
  "user": {
    "name": "git_user2",
    "username": "git_user2",
    "avatar_url": "https://blog.bobo.com.cn/s/blog_6e572cd60101qls0.html"
  },
  "project_id": 229,
  "repository": {
    "name": "rename",
    "url": "https://tencent.com:z-413/rename.git",
    "description": "is usually",
    "homepage": "https://tencent.com:z-413/rename"
  },
  "object_attributes": {
    "id": 1239,
    "note": "bug",
    "noteable_type": "Issue",
    "author_id": 23,
    "created_at": "2018-03-13T09:51:06+0000",
    "updated_at": "2018-03-13T09:51:06+0000",
    "project_id": 229,
    "attachment": null,
    "line_code": null,
    "commit_id": "",
    "noteable_id": 23,
    "system": false,
    "st_diff": null,
    "url": "https://tencent.com:z-413/rename/issues/23##note_229"
  },
  "issue": {
    "id": 8,
    "title": "test",
    "assignee_id": null,
    "author_id": 11323,
    "project_id": 229,
    "created_at": "2018-03-13T09:51:06+0000",
    "updated_at": "2018-03-13T09:51:06+0000",
    "position": 0,
    "branch_name": null,
    "description": "test",
    "milestone_id": null,
    "state": "closed",
    "iid": 66
  }
}