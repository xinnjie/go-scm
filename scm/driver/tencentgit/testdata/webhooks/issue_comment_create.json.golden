{
  "Action": "created",
  "Repo": {
    "ID": "4861503",
    "Namespace": "gitlab-org",
    "Name": "hello-world",
    "FullName": "gitlab-org/hello-world",
    "Perm": null,
    "Branch": "master",
    "Private": false,
    "Clone": "https://gitlab.com/gitlab-org/hello-world.git",
    "CloneSSH": "git@gitlab.com:gitlab-org/hello-world.git",
    "Link": "https://gitlab.com/gitlab-org/hello-world",
    "Created": "0001-01-01T00:00:00Z",
    "Updated": "0001-01-01T00:00:00Z"
  },
  "Issue": {
    "Number": 1,
    "Title": "found a bug",
    "Body": "website is broken",
    "Closed": false,
    "Author": {
      "ID": 51764,
      "Login": "sytses",
      "Name": "Sid Sijbrandij",
      "Email": "",
      "Avatar": "https://secure.gravatar.com/avatar/8c58a0be77ee441bb8f8595b7f1b4e87?s=80&d=identicon"
    },
    "PullRequest": false,
    "Created": "2017-12-10T16:43:38Z",
    "Updated": "2017-12-10T16:43:38Z"
  },
  "Comment": {
    "ID": 50771783,
    "Author": {
      "ID": 51764,
      "Login": "sytses",
      "Name": "Sid Sijbrandij",
      "Email": "",
      "Avatar": "https://secure.gravatar.com/avatar/8c58a0be77ee441bb8f8595b7f1b4e87?s=80&d=identicon"
    },
    "Body": "bump",
    "Created": "2017-12-10T16:43:38Z",
    "Updated": "2017-12-10T16:43:38Z"
  },
  "Sender": {
    "ID": 51764,
    "Login": "sytses",
    "Name": "Sid Sijbrandij",
    "Email": "",
    "Avatar": "https://secure.gravatar.com/avatar/8c58a0be77ee441bb8f8595b7f1b4e87?s=80&d=identicon"
  }
}
